content/utmt/rooms/creating new rooms/_index.md

```diff
@@ -8,15 +8,39 @@ resources:
   - name: optimalRoomSide
     src: "optimalRoomSide.png"
     title: Optimal placing for left and right transitions
+  - name: RoomSetup
+    src: "RoomSetup.png"
+    title: This is how it should look now
+  - name: RoomOnCreate
+    src: "RoomOnCreate.png"
+    title: This is how it will look when first adding a new room
+  - name: ViewsSetup
+    src: "ViewsSetup.png"
+    title: If you've done everything correctly this is how it should look 
 ---
 
 TODO: having some more screenshots in here would be nice. Also fix some phrasing.
 
 ### Preparation:
-When creating a new room, the dimensions __must__ be multiples of `320` x `240`. Having different room dimensions will cause lots of troubles. Also make sure that the room's room speed is set to `60` as that's what AM2R runs at. Entering different values will lead to the room being slowed-down or sped up.  
-Once you have your set room size, set a background by putting `bgScreenGuide` in one of the background definitions by dragging and dropping it from the left. Enable the background afterwards.  
+Before we set up tiles or textures, we must first make some preperations, to get started, create a new room by right clicking on `Rooms` in the file viewer on the left and then click add. Now you will see 3 things; 
+A list of all the ojects that are in your room
+A set of values, corresponding to the object in your room that you currently have selected
```

Comment:
```suggestion
A set of values, corresponding to the element that you currently have selected.
```
Since values apply to backgrounds/views/room name etc. too